From 36584d42a588ccf35f58532b57120e5ce0dd2180 Mon Sep 17 00:00:00 2001
From: Thierry Reding <treding@nvidia.com>
Date: Fri, 25 Apr 2014 17:10:05 +0200
Subject: [PATCH 055/389] gpio: Add missing device-managed documentation

Add the GPIO-related device-managed functions to the list of functions
in Documentation/driver-model/devres.txt.

Signed-off-by: Thierry Reding <treding@nvidia.com>
Signed-off-by: Linus Walleij <linus.walleij@linaro.org>
(cherry picked from commit f9748ef13b6a7dfe9fe9bcae8b3a8cbc10c1e188)

Signed-off-by: Tan, Raymond <raymond.tan@intel.com>
---
 Documentation/driver-model/devres.txt | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/Documentation/driver-model/devres.txt b/Documentation/driver-model/devres.txt
index 4999518..cdf014e 100644
--- a/Documentation/driver-model/devres.txt
+++ b/Documentation/driver-model/devres.txt
@@ -309,3 +309,8 @@ SLAVE DMA ENGINE
 
 SPI
   devm_spi_register_master()
+
+GPIO
+  devm_gpiod_get()
+  devm_gpiod_get_index()
+  devm_gpiod_put()
-- 
1.9.3

